pkg/skuba/actions/auth/login_test.go

```diff
@@ -325,8 +328,13 @@ func Test_Login(t *testing.T) {
 				if err == nil {
 					t.Errorf("error expected on %s, but no error reported", tt.name)
 				}
+				if err.Error() != tt.expectedErrorMsg {
+					t.Errorf("got error msg %s, want %s", err.Error(), tt.expectedErrorMsg)
+				}
```

Comment:
Suggest to change to below.

```
				if tt.expectedErrorMsg != "" {
					if !strings.Contains(err.Error(), tt.expectedErrorMsg) {
						t.Errorf("got error msg %s, want %s", err.Error(), tt.expectedErrorMsg)
					}
				}
```

Reply:
There is already a check and return.

https://github.com/SUSE/skuba/blob/5e73c11862bb4a7cc24ef3e843f798eeb16dacb8/pkg/skuba/actions/auth/login_test.go#L327-L335

Reply:
This is due to above requested change. Given that, the transitive error has no need to specify.

```
errors.Wrapf(err, "failed to query provider %s (is this the right URL? maybe missing --root-ca or --insecure, or incorrect port number?)", authReq.IssuerURL)
```